selfdrive/controls/lib/drive_helpers.py

```diff
@@ -65,6 +65,10 @@ def update_v_cruise(v_cruise_kph, buttonEvents, button_timers, enabled, metric):
         long_press = True
         break
 
+  # If set is pressed while overriding, reset cruise speed to vEgo
+  if enabled and gas_pressed and button_type == car.CarState.ButtonEvent.Type.decelCruise:
```

Comment:
Enabled is always true. See line 47
```suggestion
  if gas_pressed and button_type == car.CarState.ButtonEvent.Type.decelCruise:
```